### ISSUE TITLE
How to support the common "This will <commit message>" format?

### ISSUE THREAD

**Issue by luminarious** [egoist/changelog.md#13]
My teams in various workplaces have used this style, so I thought about how one might combine that style with this tool. How about this?

Ignore:
*	**Update** dependencies/ build scripts/ documentation
*	**Document** method/ class/ object/ function/ script/ module
* 	**Revise** linting/ code style

Patch:
*	**Fix** bug/ issue
*	**Simplify** method/ class/ object/ function/ script/ module
*	**Refactor** method/ class/ object/ function/ script/ module
*	**Optimize** method/ class/ object/ function/ script/ module

Minor:
*	**Implement** feature/ user story

Major:
*	**Remove** method/ class/ object/ function/ script/ module
*	**Evolve** feature/ API
*	**Transform** feature/ API

**Comment by egoist** [comment]
Yeah I'd like to see some improvements in this, ideally I want it to be customizable, e.g. you can provide a config file:

```js
// commit-convention.js
module.exports = {
	patch: /^(Fix|Simplify)/i
}
```
